kernel/univ.ml

```diff
@@ -471,12 +471,17 @@ struct
 	
     let super (u,n as x) (v,n' as y) =
```

Comment:
What does this return? In particular, what does `Inr b` mean? Note that I think the use of `Util.union` makes no sense in such cases. Just look at how `super` is called in `merge_univs`. There is a comment for each returned value... It should rather be explicit constructor names.